### DISCUSSION TITLE
Playback control over different devices

### DISCUSSION THREAD

**Discussion by L-Piralla** [tidal-music/discussions#68] [Ideas & Feature requests]
Hi,

I guess others have already asked for this kind of implementation but I couldn't find any related discussion in this forum.
I was wondering about the chance of controlling playback and volume on a device (let's say, my phone connected to a bluetooth speaker) from another device (like my laptop in another room), everything from the TIDAL app. I'm not talking about TIDAL connect, my question is related to android/windows/ios devices, something similar to the Spotify player.

thank you for your time! have a nice day

**Comment by stefanmoro** [comment]
Hey @pirvllv,

This GitHub discussions forum aims at handling technical discussions around the [TIDAL Developer Platform](https://developer.tidal.com/) (TIDAL API/SDK, open source projects etc).

As this question relates to our existing product offering, I will kindly ask you to reach out to our main [TIDAL support](https://support.tidal.com/).

Thanks
